src/App.tsx

```diff
@@ -3,15 +3,44 @@ import {
   ReactLocation,
   Router,
 } from "@tanstack/react-location"
-import { AppProviders } from "components/AppProviders/AppProviders"
-import { useEffect } from "react"
+import { lazy, Suspense, useEffect } from "react"
 import { routes } from "./routes"
+import HydraLogoFull from "assets/icons/HydraLogoFull.svg?react"
+import { Spinner } from "components/Spinner/Spinner"
+import { Page } from "components/Layout/Page/Page"
 
 import "unfonts.css"
 
+const AppProviders = lazy(async () => ({
+  default: (await import("components/AppProviders/AppProviders")).AppProviders,
+}))
+
 const history = createBrowserHistory()
 const location = new ReactLocation({ history })
 
+const HydraSplash = () => {
+  return (
+    <div
```

Comment:
can you create a separate styled component?

Reply:
done